portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -29,13 +32,32 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for deleting tasks. */
+/** 
+* Servlet that deletes comment entities
+*
+* @param request which contains data used to identify a comment in order to
+* delete it
+* @param response
+*/
 @WebServlet("/delete-comment")
 public class DeleteCommentServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment");
+    // Receive input on which blog we are retrieving comments from
+    int maxNumBlogs = 5;
+    int blogNumber = getUserNum(request, "blog-number", 1, maxNumBlogs);
+    if (blogNumber == -1) {
+      response.setContentType("text/html");
+      response.getWriter().println("Please enter an integer between " +  
+      1 + " to " + maxNumBlogs + ".");
+      return;
+    }
+
+    // Retrieve Comments from Datastore for the given blog post
+    FilterPredicate filterBlogComments = new FilterPredicate("blogNumber", 
+    FilterOperator.EQUAL, blogNumber);
```

Comment:
continued lines should be double-indented
```suggestion
        FilterOperator.EQUAL, blogNumber);
```